src/components/Alerts/Alert.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+type AlertProps = {
+  message: string;
+  type: 'success' | 'error';
+  duration?: number; // Optional duration for the alert to stay visible in milliseconds
+  isVisible?: boolean; // Prop to control visibility from outside
+};
+
+const Alert = ({
+  message,
+  type,
+  duration = 5000,
+  isVisible = true,
+}: AlertProps) => {
+  const [localIsVisible, setLocalIsVisible] = useState(isVisible);
+
+  useEffect(() => {
```

Comment:
Not sure about the useEffects here.
I think there is a better practice.

Reply:
Done